yew/src/virtual_dom/mod.rs

```diff
@@ -38,6 +41,10 @@ pub use self::vtag::VTag;
 #[doc(inline)]
 pub use self::vtext::VText;
 
+#[cfg(feature = "sans_mount_render")]
+#[doc(no_inline)]
+pub use self::smr::{Html, HtmlRenderError};
```

Comment:
Do we need to re-export these members? I think it would be cleaner to only make the `smr` module public (like it already is) and have other modules use the full path `use yew::virtual_dom::smr::Html`.
My goal here is to avoid feature gates as much as possible.

Reply:
This was the way I found to get `cargo check` to pick it up as connected to the rest of the code (and also rustdoc). If you have a patch that could achieve that a different way I'd be interested.

Reply:
Managed to find a solution to this.